.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,10 +18,10 @@ if the PR resolves an issue, please add further checkboxes that describe the
 action items or test scenarios from it
 -->
 
-- [ ] I have read the **CONTRIBUTING** document
-- [ ] my code follows the style requirements
+- [ ] I have read the [CONTRIBUTING.md](https://github.com/tomato42/tlsfuzzer/blob/master/CONTRIBUTING.md) document
+- [ ] my code follows the [PEP 8](https://www.python.org/dev/peps/pep-0008/) and [PEP 257](https://www.python.org/dev/peps/pep-0257/) style guides
```

Comment:
That seems something to be covered by CONTRIBUTING.md rather that another tick.

Reply:
yeah, that seems more like a generic requirement (code style) rather than something specific that everybody needs to be reminded of for every PR

I'll change it to references to CONTRIBUTING.md sections